pkg/relayer/session/session.go

```diff
@@ -261,3 +258,9 @@ func (rs *relayerSessionsManager) mapAddMinedRelayToSessionTree(
 	// Skip because this map function only outputs errors.
 	return nil, true
 }
+
+// IsWithinGracePeriod checks if the grace period for the session has ended
+// and signals whether it is time to create a claim for it.
+func IsWithinGracePeriod(sessionEndBlockHeight, currentBlockHeight int64) bool {
+	return sessionEndBlockHeight+sessionkeeper.GetSessionGracePeriodBlockCount() >= currentBlockHeight
```

Comment:
Shouldn't this be `<=`?

Reply:
It was
```go
if sessionEndblockHeight < currentBlockHeight-sessionkeeper.SessionGracePeriod` {
  return error
}
```
Which was indicating "isNOTWithingGracePeriod"

It's more readable now IMO

Reply:
See suggested change in latest review.